apps/css-workshop/src/components/Checkbox.astro

```diff
@@ -0,0 +1,61 @@
+---
+import cx from 'classnames';
+
+type Props = {
+  isLoading?: boolean;
+  label?: React.ReactNode;
+  indeterminate?: boolean;
+  checked?: boolean;
+  disabled?: boolean;
+  variant?: 'eyeball';
+  labelPosition?: 'left' | 'right';
+  status?: 'negative' | 'positive' | 'warning' | 'negative';
+} & astroHTML.JSX.HTMLAttributes;
+
+const {
+  status,
+  isLoading,
+  labelPosition = 'left',
+  label,
+  indeterminate,
+  disabled,
+  variant,
+  ...props
+} = Astro.props;
+---
+
+<label
+  class={cx('iui-checkbox-wrapper', {
```

Comment:
Instead of `cx`, you can use Astro's [`class:list`](https://docs.astro.build/en/reference/directives-reference/#classlist)

Reply:
yeah I'll make that switch